AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -16,13 +16,12 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
     @IBOutlet var versionLabel: UILabel!
     @IBOutlet var callKitOptionsPicker: UIPickerView!
     @IBOutlet var audioModeOptionsPicker: UIPickerView!
-    @IBOutlet var audioRedundancyOptionsPicker: UIPickerView!
     @IBOutlet var joinButton: UIButton!
     @IBOutlet var debugSettingsButton: UIButton!
+    @IBOutlet var audioRedundancySwitch: UISwitch!
```

Comment:
Next time we add another field we should def move all of these to the Additional Options page :p